index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <script src="./script.js" defer></script>
+  <script src="./class.js" defer></script>

```

Comment:
- [ ] JS script tag should be placed just above the closing body tag `</body>` because putting it here would slow the rendering of the HTML elements and styling during loading and re-loading. :pray:

Reply:
Apparently, defer works just as well. However, I will do as suggested.